api/templates/design.go

```diff
@@ -352,22 +353,15 @@ var _ = Service("templates", func() {
 var GenericErrorType = Type("GenericError", func() {
 	Description("Generic error")
 	Attribute("statusCode", Int, "HTTP status code.", func() {
-		Meta("struct:tag:json", "statusCode")
```

Comment:
V kode som vytvoril `UnexpectedError` pre neocakavane chyby, teda tieto `Meta` tu uz nepotrebujem.
Bude to takto lepsie, `GenericError` je spolocny format chyby pre ocakavane chyby, a `UnexpectedError` pre neocakavane + ma `exceptionId`.